base_location_geonames_import/__manifest__.py

```diff
@@ -9,22 +8,23 @@
 
 {
     'name': 'Base Location Geonames Import',
-    'version': '11.0.1.1.1',
+    'version': '12.0.1.1.1',
     'category': 'Partner Management',
     'license': 'AGPL-3',
-    'summary': 'Import better zip entries from Geonames',
+    'summary': 'Import zip entries from Geonames',
     'author': 'Akretion,'
               'Agile Business Group,'
-              'Antiun Ingeniería S.L.,'
               'Tecnativa,'
+              'AdaptiveCity,'
               'Odoo Community Association (OCA)',
-    'website': 'http://www.akretion.com',
-    'depends': ['base_location'],
-    'external_dependencies': {'python': ['requests']},
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base_location'
```

Comment:
Add always a comma at the end of the line, for reducing future diffs. It's one of the best things of Python!